pkg/rpcservice/trigger.go

```diff
@@ -32,6 +32,7 @@ func (srv *RPCService) Trigger(ctx context.Context, req *rpc.TriggerRequest) (*r
 		req.Remove,
 		req.Raw,
 		req.ApiVersion,
+		req.Edit,
```

Comment:
curious why for the fixtures rpcservice `edit` is always false, but for triggers is user defined?

Reply:
I was really just following what looked like convention here- for `triggers` we're passing the user definitions for all the params, whereas for `fixtures` we're passing the defaults